### PR TITLE
MySQL Enterprise Pack Readme Update

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Description
Adds a description for configuring non-integration event collectors.

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests - N/A
- [x] Documentation

### CODE REVIEW

File: Packs/MySQLEnterprise/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+This Pack includes XSIAM content.
+
+In order to use the collector to collect events from the vendor, you have 2 options to collect events. In either way, you will need to configure the vendor and product for this specific collector.
+1. Broker VM - You will need to use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/broker-vm/set-up-broker-vm/configure-your-broker-vm).
```
Comment (tkatzir) [inline Packs/MySQLEnterprise/README.md:4]:
```suggestion
1. Broker VM - You will need to use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/broker-vm/set-up-broker-vm/configure-your-broker-vm).\
```

File: Packs/MySQLEnterprise/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+You can configure the specific vendor and product for this instance, by going to the Settings -> Configuration -> Data Broker -> Broker VMs. Right-Click, and select Syslog Collector -> Configure.
```
Comment (tkatzir) [inline Packs/MySQLEnterprise/README.md:5]:
```suggestion
You can configure the specific vendor and product for this instance, by going to the Settings -> Configuration -> Data Broker -> Broker VMs. Right-Click, and select Syslog Collector -> Configure.\
```

File: Packs/MySQLEnterprise/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+When configuring the Syslog Collector, set vendor as mysql, product as enterprise.
+2. XDRC (XDR Collector) - You will need to use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/cortex-xdr-collectors/xdr-collector-datasets#id7f0fcd4d-b019-4959-a43a-40b03db8a8b2).
```
Comment (tkatzir) [inline Packs/MySQLEnterprise/README.md:7]:
```suggestion
1. XDRC (XDR Collector) - You will need to use the information described [here](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-pro-admin/cortex-xdr-collectors/xdr-collector-datasets#id7f0fcd4d-b019-4959-a43a-40b03db8a8b2).\
```

File: Packs/MySQLEnterprise/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+You can configure the vendor and product by replacing [vendor]_[product]_raw with mysql_enterprise_raw.
```
Comment (tkatzir) [inline Packs/MySQLEnterprise/README.md:8]:
```suggestion
You can configure the vendor and product by replacing [vendor]_[product]_raw with mysql_enterprise_raw.\
```

**Review by tkatzir — CHANGES_REQUESTED** [review]

**Comment by avidan-H** [comment]
Adding the `docs-approved` label since the README text is taken directly from the template.

**Review by tkatzir — APPROVED** [review]

**Pull request merged**